andela/travel_tool_front
# #162382329 Display progress bar on document upload

#### What does this PR do?


#### Description of Task to be completed?
- add code to display the progress bar while uploading a document
- fix error occurring when a user cancels selection after previously
  having selected an invalid document
- Modify the tests to capture the above addition.

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ft-display-progress-bar-on-document-upload-162382329** branch
   `git checkout ft-display-progress-bar-on-document-upload-162382329`
 - Install dependencies and start the server
   - `make start` *if you have docker*
   - `yarn install` then `yarn start` *if you do not have docker*
 - Start the [backend](https://github.com/andela/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
 - Click on the documents tab and add a document.
 - You should see a progress bar while the document uploads as shown in the screenshot below.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
* [#162382329](https://www.pivotaltracker.com/story/show/162382329)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/41262970/49652319-cbdf6180-fa42-11e8-9cb5-68cbd8a05e73.png)

#### Questions:
N/A
